e2e/runner.sh

```diff
@@ -33,9 +34,10 @@ fi
 ### Set sensible defaults ###
 REPO="https://github.com/apache/spark"
 IMAGE_REPO="docker.io/kubespark"
+DEPLOY_MODE="cloud"
```

Comment:
I would assume minikube is better as default as it is less assuming than GCE.

Reply:
sgtm, will flip that.

Reply:
Done